cryptoki/src/session/object_management.rs

```diff
@@ -96,6 +96,40 @@ impl Session {
         }
     }
 
+    /// Copy an object
+    /// A template can be provided to change some attributes of the new object, when allowed.
+    ///
+    /// # Arguments
+    ///
+    /// * `object` - The [ObjectHandle] used to reference the object to copy
+    /// * `template` - a reference to a slice of attributes
```

Comment:
The docs need to be improved here as it's not clear what the template is for. Random docs I found on the net clarify that:

```suggestion
    /// * `template` - new values for any attributes of the object that can ordinarily be modified
```

From: https://www.cryptsoft.com/pkcs11doc/v220/pkcs11__all_8h.html#aC_CopyObject

(Feel free to improve upon my suggestion!)

Reply:
I made an update containing your suggestion. I also added a ref to the PKCS#11 spec. That looks good for me when generating the documentation, check if that suits your taste.